Added docs on AWS with OIDC. (#5702)
harness/developer-hub: docs/continuous-delivery/deploy-srv-diff-platforms/aws/ecs/ecs-deployment-tutorial.md

1. Create or edit an IAM user and attach the custom managed policy you create above. In this example, we'll name this policy **HarnessECS**.
2. Generate an access key for the IAM user for connecting Harness to AWS ECS. In the IAM User, click **Security credentials** and then in **Access keys** create and save the key. You will use the Access key ID and Secret access key in the Harness AWS connector later.

:::note 

> If you are using the OIDC credential type for your AWS connector, also add a trust relationship between this IAM role and your identity provider. For information on using OIDC with AWS at Harness see: [AWS Connector -- Credentials](/docs/platform/connectors/cloud-providers/ref-cloud-providers/aws-connector-settings-reference.md#credentials)

Reviewer: ```suggestion
If you are using the OIDC credential type for your AWS connector, also add a trust relationship between this IAM role and your identity provider. For information on using OIDC with AWS at Harness, go to [AWS Connector -- Credentials](/docs/platform/connectors/cloud-providers/ref-cloud-providers/aws-connector-settings-reference.md#credentials)
```

> If you are using the OIDC credential type for your AWS connector, also add a trust relationship between this IAM role and your identity provider. For information on using OIDC with AWS at Harness, go to [AWS Connector -- Credentials](/docs/platform/connectors/cloud-providers/ref-cloud-providers/aws-connector-settings-reference.md#credentials).

Reviewer: ```suggestion
If you are using the OIDC credential type for your AWS connector, add a trust relationship between the IAM role and your identity provider. For information on using OIDC with AWS at Harness, go to [AWS Connector -- Credentials](/docs/platform/connectors/cloud-providers/ref-cloud-providers/aws-connector-settings-reference.md#credentials).
```

3. In **AWS Cloud Provider**, in **Name**, enter a name, and click **Continue**.
4. In Credentials, enter the following and click **Continue**:
   1. Select **AWS Access Key**.
   :::note

>    If you would like to use AWS with OIDC, select **Use OIDC** here and ignore step 3 below. 

Author: Two questions:

Is this correct?
Does this bloat the docs?

Reviewer 2: Use OIDC is another option this doesn't bloat the docs

>    If you would like to use AWS with OIDC, select **Use OIDC** here and ignore step 3 below. 

Reviewer: ```suggestion
   If you would like to use AWS with OIDC, select **Use OIDC** here and ignore Step 3 below. 
```
